src/Posts.js

```diff
@@ -1,22 +1,31 @@
-import React, { Component } from "react";
+import React from 'react'
+import axios from 'axios'
+
+export default class Posts extends React.Component {
+    state = {
+        posts: []
+    }
+
+    componentDidMount() {
+        axios.get('http://0.0.0.0:3000/posts.json')
```

Comment:
Not recommended to hardcode urls. Best to store these in the environment, or at the very least in a settings file that can be imported